testing/integration/logs_ingestion_test.go

```diff
@@ -199,14 +195,21 @@ func testMonitoringLogsAreShipped(
 
 func findESDocs(t *testing.T, findFn func() (estools.Documents, error)) estools.Documents {
 	var docs estools.Documents
+	start := time.Now()
+	defer func() {
+		fmt.Println(">>>>>>>>>>>>>>>>>>>> Took", time.Now().Sub(start))
```

Comment:
```suggestion
		t.Log(">>>>>>>>>>>>>>>>>>>> Took", time.Now().Sub(start))
```

Same applies for the one below, assuming you plan to keep these and they aren't just for debugging

Reply:
That's just for debugging, I'm still trying to understand why it only fails to find the docs in CI....